docs/content/en/blog/releases/v0.41.0.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Release v0.41.0"
+linkTitle: "Release v0.41.0"
+date: 2022-12-20
+description: >
+ Release v0.41.0
+---
+
+## Changes since v0.40.1
+
+### Breaking Changes
+
+All __applications configuration__ file should NOT use the `spec.triggerPaths` (marked as deprecated previously), but use `spec.trigger.onCommit.paths` instead.
+
+* Remove trigger paths ([#4037](https://github.com/pipe-cd/pipecd/pull/4037))
+
+### New features
+
+From now, you can manage your project's deployment via PipeCD deployment metrics. Please refer to the [docs](/docs/user-guide/insights/).
```

Comment:
@khanhtc1202 I'm not sure about this `manage project's deployment`. 🤔

Reply:
Got it. How about `From now, you can view the deployment insights on your application and project. Please refer to ..`

Reply:
```
From now, you can have a better insight into your application/project's deployment via deployment data visualized by the PipeCD insight feature. Please refer to the [docs](/docs/user-guide/insights/).
```
thanks for comment, I think I will go with this 👀